### ISSUE TITLE
Upcoming changes to API and impact on python-twitter

### ISSUE THREAD

**Issue by charlesreid1** [bear/python-twitter#561]
As you've probably seen, there are some changes in the works for the twitter API, as mentioned in [this announcement](https://blog.twitter.com/developer/en_us/topics/tools/2017/announcing-more-functionality-to-improve-customer-engagements-on-twitter.html) around a year ago:

> on Tuesday June 19, 2018 we are retiring the following services and endpoints:
> 
>     User Streams
>     Site Streams
>     GET direct_messages
>     GET direct_messages/sent
>     GET direct_messages/show
>     POST direct_messages/new
>     POST direct_messages/destroy

Was just wondering what impact this will have, if any, on python-twitter functionality and if there are plans in the works for an update?

**Comment by charlesreid1** [comment]
oops, that's the wrong link, I _can_ do date math. let me see if I can find the right one. [here we go](https://blog.twitter.com/developer/en_us/topics/tools/2017/building-the-future-of-the-twitter-api-platform.html).

**Comment by jeremylow** [comment]
Yep, started a Project on this. The documentation is a little all over the place and has changed considerably since it was first announced. I'm going back through it now.

**Comment by BoraxTheClean** [comment]
Are there any updates on this? I'm particularly interested in the user streams functionality. Do you guys need help?

The sunset is now Thursday, August 16, 2018.

**Comment by webtweakers** [comment]
The deadline has passed, yet no updates. Any news? Need help?
